botCommands/__snapshots__/debug.test.js.snap

```diff
@@ -1,3 +1,18 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`/debug callback should return the correct output 1`] = `"Looks like this problem can be solved using a debugger: <https://developers.google.com/web/tools/chrome-devtools/javascript>"`;
+exports[`/debug js should return the correct output 1`] = `
+"Based on the description of your problem, you can get to the root of it using a debugger. Learning how to track down problems like this is an inevitable part of being a developer.
+<https://developers.google.com/web/tools/chrome-devtools/javascript>"
+`;
+
+exports[`/debug rb should return the correct output 1`] = `
+"Based on the description of your problem, you can get to the root of it using a debugger. Learning how to track down problems like this is an inevitable part of being a developer.
+<https://www.theodinproject.com/courses/ruby-programming/lessons/debugging#debugging-with-pry-byebug>"
+`;
+
+exports[`/debug should return the correct output 1`] = `
+"Based on the description of your problem, you can get to the root of it using a debugger. Learning how to track down problems like this is an inevitable part of being a developer.
+<https://en.wikipedia.org/wiki/Debugging>
+
+To get a helpful resource in javascript or ruby, run \`/debug js\` or \`/debug rb\`."
```

Comment:
Let's capitalize "Javascript" and "Ruby" and say "To get a language-specific resource..." instead.